README.md

```diff
@@ -70,4 +70,6 @@ Give a ⭐️ if you like this project!
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [OPTIONAL] When I click the link to the `MIT.md` file that you added, I get this 404 `error`:

![00Mit](https://user-images.githubusercontent.com/79058364/182822618-fdfb2523-cb23-4060-9f72-75ec8c29e4cd.png)


#### The reason for this is that there is no `MIT.md` file in the root directory of this project. Kindly attach the file. 👍